CHANGELOG.md

```diff
@@ -1,5 +1,105 @@
 # Changelog
 
+## [0.60]
+
+PS: contributing guide has been updated! ([a6f905f](https://github.com/facebook/react-native/commit/a6f905f) by [@hramos](https://github.com/hramos))
```

Comment:
maybe this could simply be an extra line in the changed section? It feels a bit weird to have a PS at the start of a text (it should be at the end)

Reply:
Also, maybe here at the top wee should have something like a brief intro text, something between this
```
This is a patch fix release addressing a couple ScrollView regressions, and "future-proof" RN 59 from crashes caused by upgrading Gradle (now can support up to 5.4.1 & 3.4.0 for the plugin) and Xcode 11 Beta 1. You can upgrade to this version without upgrading your tooling.
Thanks everyone who contributed code or participated in the [discussion](https://github.com/react-native-community/react-native-releases/issues/124) for cherry-picking commits.  You can participate to the conversation for the next patch release in the dedicated [issue](https://github.com/react-native-community/react-native-releases/issues/127).
```

(copied from 0.59.9)

and the higlights we posted for the releases tab -> https://github.com/facebook/react-native/releases/tag/v0.60.0-rc.3

Reply:
Yeah, I still need to write the summary at the top. This will be part of the summary.